### ISSUE TITLE
⌨ Inconsistent use of semantic markup to mark emphasized or special text

### ISSUE THREAD

**Issue by MaddyDaigle** [GCTC-NTGC/design-gc-digital-talent#22]
## ⌨ Accessibility Issue

Bolded text on the website is coded with css styles instead of semantic HTML markup. This does not demonstrate to the screenreader that these sections are emphasized.

### WCAG 2.1 Criteria

H49: Using semantic markup to mark emphasized or special text
REF: https://www.w3.org/TR/WCAG20-TECHS/H49.html

## 🦋 Expected Behaviour

Stanford University and WAI research suggests that inconsistent emphasis and overuse of the bold style can make it hard for the user to understand any of your content, let alone what is important for them to retain.

Cognitive accessibility researchers have also found that bolded text should be used to describe key words and phrases (1-3 words) for memory retention.

If one needs to bold or italicize full sentences or paragraphs, it might be best to use bullets.

### Acceptance Criteria

A set of assumptions which, when tested, verify that the accessibility requirement was met.

- [ ] List of places that we use too much bolding.
- [ ] Use the strong convention to emphasize clear, key words that draw user attention to important points or concepts within those sentences.

**Comment by gobyrne** [comment]
@RM-1978 can we address this as part of the copy review process? That's really the only place where we'll be auditing the whole of our content, so probably best to include it there. If you agree, please close this issue as the copy review is tracked elsewhere.

From the above discussion, we're mostly in agreement about the <strong> tag and I think @RM-1978 guidance on just bolding 1-3 words at a time is sound.

Where this does not apply is where bolding is used to visually differentiate different components like the ones @substrae listed or these key/value pairs that we have everywhere on the profile.

![image](https://github.com/GCTC-NTGC/design-gc-digital-talent/assets/2494551/d0a71a84-7913-493a-91cd-2fdee5d602c7)

**Comment by gobyrne** [comment]
Ok, we did look at this for the application process (a little). But the issue is stale. I'm gonna close it. We can reopen, but new issues should focus on fixing specific places where this is failing.

**Comment by RM-1978** [comment]
@gobyrne it might also be good to  document how you might like to use semantic markup for emphasized or special text in specific designs.
